### PR TITLE
Texture update 2

### PR DESCRIPTION
**Pull request by Litorom**
Reverts majority of the textures and fixes stone door bug.

### CODE REVIEW

File: src/client/resources/assets/textures/item/spawner.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/spawner.png:1]: The old texture does not meet the entity form. It looks swollen.

File: src/client/resources/assets/textures/item/stone_wall.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/stone_wall.png:1]: The current one looks more like a wall, as well as plank wall.

File: src/client/resources/assets/textures/item/white_flower.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/white_flower.png:1]: The current one is more closed to white.

File: src/client/resources/assets/textures/tile/oak.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/tile/oak.png:1]: Is this not included in the tree variant update?

File: src/client/resources/assets/textures/item/gunpowder.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/gunpowder.png:1]: The current one adds more details.

File: src/client/resources/assets/textures/item/green_wool.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/green_wool.png:1]: Hmm, why especially green wool?

File: src/client/resources/assets/textures/item/cloud_ore.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/cloud_ore.png:1]: The old one is worse.

File: src/client/resources/assets/textures/item/chest.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/chest.png:1]: There is a change in light source. I think the current one is better.

File: src/client/resources/assets/textures/item/cactus.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/cactus.png:1]: I am not sure about this.

File: src/client/resources/assets/textures/item/book.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/book.png:1]: I think vertical should be better.

File: src/client/resources/assets/textures/item/bed.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/bed.png:1]: Vertical as well as beds.

**Comment by Litorom** [comment]
As the one who made the textures what they are, you have no jurisdiction

**Comment by BenCheung0422** [comment]
Whatever it is, I can still have opinions on it. Some textures like books and beds can represent better in vertical looks. That is one of what I think, is it not?

File: src/client/resources/assets/textures/item/gunpowder.png
Comment (TheBigEye) [reply src/client/resources/assets/textures/item/gunpowder.png:1]: Yeah, is true

File: src/client/resources/assets/textures/item/cactus.png
Comment (TheBigEye) [reply src/client/resources/assets/textures/item/cactus.png:1]: The new one seems to be a brighter green, it makes sense with the green wool

File: src/client/resources/assets/textures/item/chest.png
Comment (Makkkkus) [reply src/client/resources/assets/textures/item/chest.png:1]: I actually agree with this, the new one is a bit better.

File: src/client/resources/assets/textures/item/gunpowder.png
Comment (Makkkkus) [reply src/client/resources/assets/textures/item/gunpowder.png:1]: I agree with this one as well.

**Review by Makkkkus — DISMISSED** [review]
I agree with this, although two textures pointed out do have good reasons to why they changed.

File: src/client/resources/assets/textures/item/chest.png
Comment (BenCheung0422) [reply src/client/resources/assets/textures/item/chest.png:1]: Chest as well as dungeon chest.

File: src/client/resources/assets/textures/entity/bed.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/entity/bed.png:1]: Actually I think it looks smaller.

File: src/client/resources/assets/textures/item/plank_wall.png
Comment (BenCheung0422) [inline src/client/resources/assets/textures/item/plank_wall.png:1]: The current occupying 8*8 might not good, but I think the old one does not meet the color scheme of planks.

File: src/client/resources/assets/textures/item/white_flower.png
Comment (BenCheung0422) [reply src/client/resources/assets/textures/item/white_flower.png:1]: I think the green shadow looks incorrect on white petal.

**Pull request merged**